spec/requests/providers/means/student_finances_controller_spec.rb

```diff
@@ -141,20 +135,6 @@
           expect(response).to redirect_to(next_page)
         end
       end
-
-      context "when the provider does not have bank statement upload permissions" do
```

Comment:
is this path/flow not ppossible now for truelayer? 🤷

Reply:
:+1: 
I removed tests based on permissions - I should have refactored into using `uploading_bank_statements?`